ansible/README_DISTRIBUTED.md

```diff
@@ -0,0 +1,110 @@
+Deploying Distributed OpenWhisk using Ansible
+=========
+
+### Distributed Deployment using Ansible
+**Caveat:** All Ansible commands are meant to be executed from the `ansible` directory.
+This is important because that's where `ansible.cfg` is located which contains generic settings that are needed for the remaining steps.
+
+By default, if you omit the `-i` parameter in ansible commands, the `local` environment will be used.
+
+In all instructions, replace `<openwhisk_home>` with the base directory of your OpenWhisk source tree. e.g. `openwhisk`
+
+Login to your bootsrapper VM. Your local machine can act as bootstrapper as well, if it can connect to other VMs.
+
+Install prereqs for ansible
+```
+sudo apt-get -y install python-pip python-setuptools python-dev libssl-dev
+```
+
+Install Ansible and cloud module packages:
+```
+sudo pip install ansible==2.0.2.0 markupsafe shade pitz positional appdirs monotonic rfc3986 jsonschema
```

Comment:
Why are so many prereqs needed to be installed explicitly? It seems to me the `os_server` module only needs `shade`. What's the rest for?